presto-kudu/src/main/java/io/prestosql/plugin/kudu/KuduClientSession.java

```diff
@@ -500,20 +500,31 @@ else if (domain.isSingleValue()) {
                     }
                     else if (valueSet instanceof SortedRangeSet) {
```

Comment:
Do you have an example of speedup you got with this change?

Reply:
Itest with a table with 400 million rows, queries like SELECT * FROM kudu_table WHERE str_column IN (...); could be reduced from 47 seconds to 3 secends.  Rows presto scaned is reduced from hundred millions to  the size of in-list